addons/calendar/models/calendar_event.py

```diff
@@ -180,7 +180,7 @@ def _default_partners(self):
         ('self_only', "This event"),
         ('future_events', "This and following events"),
         ('all_events', "All events"),
-    ], store=False, copy=False, default='self_only',
+    ], store=False, copy=False,
```

Comment:
seems odd to add a new "unset" value, what is it meant to be doing?

before the change, there was no empty possible value (from the interface) now there is. also if you don't change the selection in the interface you see this (and when saving `recurrence_update` will be unset):

> ![2024-02-01-205855_256x188_scrot](https://github.com/odoo/odoo/assets/9977887/b0c0e24d-1c94-4fd3-bf6a-17b6d03c3c3f)

but if you click on "All events" then "This event", the recurrence_update will be set to `self_only` but without any change in the interface. This seems unexpected.